### PR TITLE
Update md files

### PR DESCRIPTION
**Pull request by perlajarillo**
In this branch we are updating the information about the program in the README and LICENSE files.

### CODE REVIEW

**Comment by perlajarillo** [comment]
Related with #58

File: LICENSE.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 MIT License
 
 Copyright (c) 2019 Perla Jarillo, Faviola Vieyra
```
Comment (Fa-v) [inline LICENSE.md]: *Fabiola

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
-# FLAD-Mentorship app
+# Let's Talk About Business app (before FLAD-Mentorship)
 
-FLAD Mentorship helps to link people to participate as a mentor or a mentee in the program [Let's talk about business](https://www.flad.pt/en/lets-talk-about-business/) from the Luso-American Development Foundation.
+This app helps to link people to participate as a mentor or a mentee in the program [Let's talk about business](https://www.flad.pt/en/lets-talk-about-business/) in coordination to the Flad-Mentorship program, sponsored by the Luso-American Development Foundation.
```
Comment (Fa-v) [inline README.md]: *in coordination with...

File: README.md [hunk 3]
```diff
@@ -31,7 +31,7 @@ To enable Email/Password as authentication method in your Firebase project:
 
 ### 4. Create a folder for images
 
-Next you will need to create a folder to storage the pictures of the mentors participating in FLAD-Mentorship program.
+Next you will need to create a folder to storage the pictures of the mentors participating in Let's Talk About Business program.
```
Comment (Fa-v) [inline README.md]: Next, you will need to create a folder to *store...

**Review by Fa-v — CHANGES_REQUESTED** [review]
Hi Perla, We just need to fix some little things :)

File: LICENSE.md (hunk 1 above)
Comment (perlajarillo) [reply LICENSE.md]: I am so sorry 🙈

**Comment by perlajarillo** [comment]
I think I made all the requested changes, do you see something else we can improve @Fa-v ?

File: LICENSE.md (hunk 1 above)
Comment (Fa-v) [reply LICENSE.md]: Don't worry :)

**Review by Fa-v — APPROVED** [review]

**Pull request merged**